acceptance_tests/features/view_survey_list.feature

```diff
@@ -0,0 +1,19 @@
+Feature: View survey list
+  As a Respondent
+  I need to see collection exercise(s) only if the the event 'Go-Live' has occurred/passed
+  So that collection exercises are not visible to respondents before they should be
+
+  Background: User already logged in
+    Given the respondent is signed into their account
+
+  @us060-filterCEByGoLive_s01
+  Scenario: Respondent can view collection exercise after the Go Live Event
```

Comment:
If you sign in with the background you will need to add a then to the scenarios to sign out otherwise any following tests requiring sign in will fail

Reply:
We need to refector the login/logout stuff into the before_step and after_step methods in behave. They should be assumptions for most tests

Reply:
We could use 

```python
def after_step():
    browser.cookies.delete() 
```

Reply:
@rhi The tests still pass here because you can still redirect to the sign-in page in frontstage while logged in
I agree that this is a better way to do these things

Reply:
Behave methods document using the background method for functionality like sign in.

Reply:
@rhiGriff fair point for the login actions but the logout step is just a hack to fix that sessions are leaked across each test so I think the after step is valid

Reply:
We have 2 separate log out functionality for internal and external how would we go about this? Happy to discuss further

Reply:
@rhiGriff @benjefferies 
This isn't causing failing tests for this story + is left over from a previous PR.
Can this issue be fixed in another PR so we can move this story in Business Review